src/kafka.rs

```diff
@@ -75,6 +75,8 @@ pub async fn run_async_processor(worker_id: usize, sr_settings: SrSettings) -> R
 
     let consumer: StreamConsumer = create_consumer()?;
     let mut decoder = AvroDecoder::new(sr_settings);
+    let score_definitions = ScoreGraph::new()?.scores()?;
```

Comment:
Hvorfor bruker du ikke scoreGraph her istedet for scores?

Reply:
`ScoreGraph` brukes til å lese og parse scores, og output'er en `ScoreDefinition`. Etter å ha beregnet `ScoreDefinition` én gang er ikke `ScoreGraph` nødvendig lenger.